test/Handler/CommonSpec.hs

```diff
@@ -11,7 +11,7 @@ spec = withApp $ do
         it "has correct User-agent" $ do
             get RobotsR
             bodyContains "User-agent: *"
-    describe "favicon.ico" $ do
+    describe "favicon.ico" $
```

Comment:
I'd probably ignore this hlint advice, as this still makes it harder to add tests in the future with minimal line-diffs.